app/static/js/picslider.js

```diff
@@ -1,16 +1,11 @@
-var svg = d3.select("#face_view_svg");
-let svg_width=svg.node().getBoundingClientRect().width;
-let svg_height=svg.node().getBoundingClientRect().height;
-
-let margin = {top: 20, right: 20, bottom: 110, left: 40};
-let margin2 = {top: 30, right: 20, bottom: 50, left: 40};
-
-let width = svg_width - margin.left - margin.right;
-let height = svg_height ;
-let height2 = svg_height;
+var width = 750;
```

Comment:
I think we already have these names in another file. So if sizing causes trouble then look at these variable names and change them.

Reply:
these should apply to the viewbox afaik - so it's only a width inside the svg right?